### PR TITLE
Makes the navbar similar to google

### LINKED ISSUE #1398: Make navbar of results page similar to google.com
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

- Make navbar of results page similar to google
- Add tests for the same

**Screenshots of the issue:**

screenshot from 2019-01-10 00-04-45

screenshot from 2019-01-10 00-05-51

**Will work on the issue:**
Yes

### PR DESCRIPTION
**Pull request by rajvaibhavdubey**
Fixes #1398 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- The search bar has been rounded
- The background of the navbar changed to white
- A seperate component has been created for the same because on the homepage the original search-bar is being used.

- https://pr-1401-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by subhahu123** [comment]
i there is no need to create component for that because they are same except that now bar is round 
we can just create a ng-classs where we can use round css for result search (round bar) and one for index bar ( corner ) . i find no to create a seperate component for this ..as this is not required for just create a bar round @rajvaibhavdubey

**Comment by subhahu123** [comment]
moreover there is  a problem with top-left corner
screenshot-pr-1401-fossasia-susper surge sh-2019 01 11-22-07-51
``

**Review by shreyanshdwivedi — COMMENTED** [review]
I also don't find any specific reason for use of new component for making the `search-bar` rounded. 
Also, I'll suggest not to use white background only for search-bar. Instead use same background color for the `search-bar` as well as `search-options-field` just like `Google`.
screenshot_2019-01-11 susper - decentral search engine

**Comment by rajvaibhavdubey** [comment]
@subhahu123 @shreyanshdwivedi I created a different component because we need two types of search-bar , similar to google(rectangular on index and rounded on results page). I could'nt find a way to do the same without creating a new component. If you know one, do tell me :sweat_smile: .
@shreyanshdwivedi Will do the changes .

**Review by subhahu123 — CHANGES_REQUESTED** [review]
fix top left corner

**Review by subhahu123 — CHANGES_REQUESTED** [review]
screenshot_2019-01-22 susper - decentral search engine
still not working

**Comment by rajvaibhavdubey** [comment]
@subhahu123 I have not fixed it yet, Please wait.

**Comment by rajvaibhavdubey** [comment]
@shreyanshdwivedi @subhahu123 @praveenojha33 Why is the test covergae decreasing for the components I haven't made changes to like intelligence, navbar, results.

**Review by praveenojha33 — COMMENTED** [review]
@rajvaibhavdubey I think those components are connected with the components you made changes.

File: src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <nav
   class="top-nav navbar navbar-static-top navbar-default"
-  [style.background-color]="themeService.navbarbgColor"
+  style="background-color:white"
```
Comment (sakshee-19) [inline src/app/navbar/navbar.component.html:3]: Move it to css file.

File: src/app/search-bar-results/search-bar-results.component.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  @media screen and (max-width: 811px) and (min-width: 768px) {
+    #nav-input{
+      width: 60vw;
+    }
+    #nav-group{
+      width: 60vw;
+    }
+  }
+  
+
```
Comment (sakshee-19) [inline src/app/search-bar-results/search-bar-results.component.css:99]: Remove extra lines.

File: src/app/search-bar-results/search-bar-results.component.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  @media screen and (max-width: 360px) {
+    .align-search-btn{
+      left: -1%;
+    }
+  
+    #nav-input {
+      width: 63vw;
+    }
+  }
+
```
Comment (sakshee-19) [inline src/app/search-bar-results/search-bar-results.component.css:145]: Add EOF.

File: src/app/search-bar-results/search-bar-results.component.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  <script>
+    $(document).ready(function() {
+      var isFirefox = typeof InstallTrigger !== 'undefined';
+  
+      if(isFirefox === false){
+        $(".input-group-btn").addClass("align-search-btn");
+      }
+    })
+  </script>
+
```
Comment (sakshee-19) [inline src/app/search-bar-results/search-bar-results.component.html:35]: Add EOF.

**Review by sakshee-19 — CHANGES_REQUESTED** [review]
Please do some changes.

**Comment by simsausaurabh** [comment]
Closing due to inactivity, please follow up with latest codebase.

**Pull request closed without merging**